tests/test_cli.py

```diff
@@ -225,3 +225,21 @@ def test_run_uncaptured(self, capfd):
             # Check that the expected output is somewhere between the conda logs
             captured = capfd.readouterr()
             assert random_text in captured.out
+
+    def test_run_readonly_env(self):
+        from tests.test_create import make_temp_env
+        from tests.test_create import make_temp_prefix
+        import stat
```

Comment:
Would you mind moving those to the top of the file, please?

Reply:
I'm happy to move the `import stat` to the top of the file, but the other non-toplevel imports I'm following the pattern in the rest of the file.  I assumed they were put there for a reason, but I haven't investigated.